persona/client/index.js

```diff
@@ -6,7 +6,7 @@ var jwt = require("jsonwebtoken");
 var CacheService = require("cache-service");
 var uuid = require('uuid');
 
-var clientVer = require('../package.json').version || 'unknown';
+var clientVer = require('../../package.json').version || 'unknown';
```

Comment:
So previously this was the version of the `persona-node-client` which was `4.1.6`, this will now be `0.0.1`. Will this cause us issues going forwards when we start to reuses version numbers previously deployed in `persona-node-client`. Should we skip straight to `5.0.0` with this?

Reply:
Not sure I'm clear on why re-using version numbers will be an issue. Isn't this a new library, talis-node, which starts at 0.0.1? Why will version numbers conflict?

Reply:
@malcyL it is used as part of the user agent: https://github.com/talis/talis-node/blob/master/persona/client/index.js#L79

So we'll have already seen user agents with the version 4.1.6 etc, and at some point we are likely to reuse version numbers. Not sure if that will cause any unintended consequences.

Reply:
This won't cause any issues as the agent string is only used for tracing who is calling us.

Reply:
It is worth changing the agent string to reflect the new package name though. i.e. talis-node

Reply:
I have now updated the user agent from `persona-node-client` to `talis-node`